gulpfile.babel.js

```diff
@@ -54,29 +53,36 @@ var webpackConfig = {
         test: /\.jsx|.js$/,
         exclude: /node_modules\//,
         loaders: [
-          'babel-loader?stage=1&plugins[]=object-assign'
-        ]
+          'babel-loader?stage=1&plugins[]=object-assign',
+        ],
       },
     ],
     postLoaders: [
       {
-        loader: "transform/cacheable?brfs"
-      }
-    ]
+        loader: "transform/cacheable?brfs",
+      },
+    ],
   },
   resolve: {
-    extensions: ['', '.js', '.jsx']
+    extensions: ['', '.js', '.jsx'],
   },
-  plugins: gulpPlugins
+  plugins: gulpPlugins,
 };
 
-gulp.task('test-unit', function (done) {
+gulp.task('lint', function() {
+  return gulp.src([paths.gulpfile, paths.src, paths.example])
+             .pipe(plugins.eslint())
```

Comment:
The [docs](https://github.com/adametry/gulp-eslint#ignore-files) suggests that it does on entry to a folder, but I had to add it explicitly for it to pick up the eslintignore file